pkg/flare/archive_win.go

```diff
@@ -208,6 +214,8 @@ func exportWindowsEventLog(eventLogChannel, eventLogQuery, destFileName, tempDir
 	return err
 }
 
+type filePermsInfo struct{}
```

Comment:
Dummy question: why do we need this ? I can't see where it's used 👨‍🦯

Reply:
It's used in [archive.go](https://github.com/DataDog/datadog-agent/pull/14065/files/4f9ba51f626e6c0b46634d83ecd541eeec1642f8#diff-ef51500f6ef43587bb90cdf6a09d1e200d8e1d82d399537416047495608fc693L75) to collect the permissions on the files in the flare. But the permission collecting [code isn't implemented](https://github.com/DataDog/datadog-agent/pull/14065/files/4f9ba51f626e6c0b46634d83ecd541eeec1642f8#diff-98898a0f4a202f71341bed2eeed3050173bbc1ada360d6a0ac71c0b2528d98e6R219) on Windows.